counsel.el

```diff
@@ -1678,6 +1679,57 @@ When INITIAL-INPUT is non-nil, use it in the minibuffer during completion."
           (format "http://debbugs.gnu.org/cgi/bugreport.cgi?bug=%s"
                   (substring url 1)))))))
 
+(defvar counsel-url-expansions
+  nil
+  "List of (REGEXP . FORMAT) pairs.
```

Comment:
I suggest a summary line slightly more descriptive and suggestive of the variable's purpose, e.g. "Map URL regexps to their expansion format string" or similar. The datatype structure can then be further described in the docstring body.

Another nitpick is that the variable's initial value, `nil`, could be placed on the same line as the variable's name.

Reply:
ok